src/System.Data.Common/tests/System/Data/DataColumnTest.cs

```diff
@@ -992,7 +855,7 @@ private DataColumn MakeColumn(string col, string test)
                 Expression = test
             };
         }
-
+        // TODO: what's this?
```

Comment:
Why is this disabled?

Reply:
Don't know.  This file of tests is from Mono.  It's been ifdef'd out like this since we brought it into the repo.  It should have been deleted when that happened... I must have just missed it.  Please feel free to delete it (and any other such cases in these tests).